src/js/components/awardv2/contract/ContractContent.jsx

```diff
@@ -57,6 +62,17 @@ const ContractContent = ({
         jumpToSection("award-history");
     };
 
+    useEffect(() => {
+        if (searchSubAwardTableClicked) {
+            jumpToSubAwardHistoryTable();
+        }
+    });
```

Comment:
We should only need to call jumpToSubAwardHistoryTable on didMount.

Also, we should only need to remove the property in redux on didUnMount.

My understanding, this should be one useEffect:

```javascript
useEffect(() => {
        if (searchSubAwardTableClicked) {
            jumpToSubAwardHistoryTable();
        }

        return () => updateSubAwardTable(false);
    }, [searchSubAwardTableClicked, jumpToSubAwardHistoryTable, updateSubAwardTable]);
```